integr8ly/keycloak-operator
# upgrade logic from 7.2 to 7.3

## Motivation
Upgrade from 7.2 sso to 7.3


## Verification Steps

- provision a 1.4.1 cluster and login with the customer-admin
- Import the sso 7.3 image stream ``` oc -n openshift import-image redhat-sso73-openshift:1.0 ``` done by installer for upgrade
- Login with the admin user and update the existing keycloak operator to this PR image build
- Ensure the SSO instance is scaled down redeployed and scaled up
- Ensure you can still login to the openshift cluster and the various components


## Checklist:
